benchmarks/dummy/src/main/java/org/renaissance/dummy/DummyTeardownFailing.java

```diff
@@ -1,25 +1,25 @@
 package org.renaissance.dummy;
 
-import org.renaissance.Config;
-import org.renaissance.License;
-import org.renaissance.RenaissanceBenchmark;
+import org.renaissance.Benchmark;
+import org.renaissance.BenchmarkContext;
 import org.renaissance.BenchmarkResult;
-import org.renaissance.SimpleResult;
+import org.renaissance.BenchmarkResult.Validators;
+import org.renaissance.License;
 
 import static org.renaissance.Benchmark.*;
 
 @Name("dummy-teardown-failing")
 @Group("dummy")
 @Summary("A dummy benchmark for testing the harness (fails during teardown).")
 @Licenses(License.MIT)
-public final class DummyTeardownFailing extends RenaissanceBenchmark {
+public final class DummyTeardownFailing implements Benchmark {
```

Comment:
If somebody runs all of the benchmarks together, will this fail the everything? Or just leave a note about the failure in the result files.

Reply:
It would only fail the one benchmark but dummy benchmarks are excluded from the "all" group.